applications/IgaApplication/custom_modelers/nurbs_geometry_modeler.h

```diff
@@ -0,0 +1,104 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Manuel Messmer
+//
+
+#if !defined(KRATOS_NURBS_GEOMETRY_MODELER_H_INCLUDED )
+#define  KRATOS_NURBS_GEOMETRY_MODELER_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "modeler/modeler.h"
+#include "geometries/nurbs_volume_geometry.h"
+#include "geometries/nurbs_shape_function_utilities/nurbs_volume_utilities.h"
+#include "includes/node.h"
```

Comment:
```suggestion
```

Reply:
probably worth a try but it might be needed as the geometry is templated by node

Reply:
`model_part.h` is included in the next line which includes ofc also the node

just saying that usually if model_part is included we don't include node, element etc too, but this is up to you :D

Reply:
Removed it.

Reply:
sweet